### PR TITLE
Allow to use the browserLanguage in custom templates

### LINKED ISSUE #1620: provide the browserLanguage in the Angular scope
We should provide the browser language in the Angular scope.
This helps to customize html templates.

In app.js we should add something like this:

            $rootScope.browserLanguage = browserLanguage;

### PR DESCRIPTION
**Pull request by cornelinux**
Closes #1620

### CODE REVIEW

File: privacyidea/static/components/token/views/token.list.html [hunk 1]
```diff
@@ -11,6 +11,15 @@
 
 <span translate>total tokens:</span> {{ tokendata.count }}
 
+Browserlanguge {{ browserLanguage }}
```
Comment (plettich) [inline privacyidea/static/components/token/views/token.list.html]: Is there a reason why this is in the productive html?

Comment (cornelinux) [reply privacyidea/static/components/token/views/token.list.html]: Yeah. I did mean to do this.

It is there to be found by you! ;-)

File: doc/faq/customization.rst [hunk 2]
```diff
@@ -85,6 +108,23 @@ A good stating point might be the themes at http://bootswatch.com.
...
+You can also use the substitute module of the Apache webserver.
+It is not clear how much performance impact you get, since this
+module can scan and replace all texts that is delivered by the web server.
+
+If you for example want to replace the title of the webpages, you could
+do it like this::
+
+       <Location "/">
+           AddOutputFilterByType SUBSTITUTE text/html
+           Substitute "s/>privacyidea Authentication System</>My own 2FA system</ni"
```
Comment (plettich) [inline doc/faq/customization.rst]: Not sure whether the substitute module is case sensitive but the original title is "privacyIDEA Authentication System" (See [`templates/header.html`](https://github.com/privacyidea/privacyidea/blob/0a21fd13a422a45dfe64cafb1f996bff06c188be/privacyidea/static/templates/header.html#L29))

File: doc/faq/customization.rst [hunk 3]
```diff
@@ -85,6 +108,23 @@ A good stating point might be the themes at http://bootswatch.com.
 .. note:: If you add your own CSS file, the file *bootstrap-theme.css* will
    not be loaded anymore. So you might start with a copy of the original file.
 
+
+Use web server substitute module
+................................
+
+You can also use the substitute module of the Apache webserver.
+It is not clear how much performance impact you get, since this
+module can scan and replace all texts that is delivered by the web server.
```
Comment (fredreichbier) [inline doc/faq/customization.rst]: Typo: *that are

File: doc/faq/customization.rst [hunk 4]
```diff
@@ -85,6 +108,23 @@ A good stating point might be the themes at http://bootswatch.com.
 .. note:: If you add your own CSS file, the file *bootstrap-theme.css* will
    not be loaded anymore. So you might start with a copy of the original file.
 
+
+Use web server substitute module
```
Comment (fredreichbier) [inline doc/faq/customization.rst]: Hm, even though the solution using the substitute module works, it seems a bit hacky :) What do you think about adding a more "official" way to change the title in 3.1? I can add a issue for that.

File: privacyidea/static/components/token/views/token.list.html (hunk 1 above)
Comment (cornelinux) [reply privacyidea/static/components/token/views/token.list.html]: done in 262444ca

File: doc/faq/customization.rst (hunk 4 above)
Comment (cornelinux) [reply doc/faq/customization.rst]: Yes, please do it.

File: doc/faq/customization.rst (hunk 2 above)
Comment (cornelinux) [reply doc/faq/customization.rst]: The is configured to be case insensitive (the **i** at the end).
Nice as example.
https://httpd.apache.org/docs/2.4/mod/mod_substitute.html#page-header

File: doc/faq/customization.rst (hunk 3 above)
Comment (cornelinux) [reply doc/faq/customization.rst]: will do

Comment (cornelinux) [reply doc/faq/customization.rst]: done in e76d9717

File: doc/faq/customization.rst (hunk 4 above)
Comment (fredreichbier) [reply doc/faq/customization.rst]: I've opened #1624.

**Pull request merged**